src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -85,6 +85,13 @@ public static void ThrowJsonException(string? message = null)
             throw ex;
         }
 
+        [DoesNotReturn]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public static void ThrowInvalidOperationException_CannotSerializeOpenGeneric(Type type)
```

Comment:
Do we have tests for properties that return open generics?

Reply:
such object definitions give an error "Unexpected use of an unbound generic name"

```c#
private class Class
{
    public List<> MyList { get; set; }
}
```

Can one make a type like this using reflection?

Reply:
I was thinking `MyList<T>` for `Class<T>` but that would be caught by the check for `Class<T>`.